ui/src/app/pages/access/access-renewal.spec.tsx

```diff
@@ -27,7 +27,7 @@ describe('Access Renewal Page', () => {
     const {profile} = profileStore.get();
 
     const newProfile = fp.set('accessModules',
-        {modules: expirableAccessModules.map(m => {return {moduleName: m, expirationEpochMillis: expiredTime}})},
+        {modules: Array.from(accessRenewalTitles.keys()).map(m => {return {moduleName: m, expirationEpochMillis: expiredTime}})},
```

Comment:
Optional, existing issue; would definitely avoid 1-liner with a return statement. Inline object expressions can be used with arrow functions, you just need to wrap the object in parens.

```suggestion
        {modules: Array.from(accessRenewalTitles.keys()).map(m => ({moduleName: m, expirationEpochMillis: expiredTime}))},
```